Library/Formula/dcmtk.rb

```diff
@@ -37,6 +45,6 @@ def install
   test do
     system bin/"pdf2dcm", "--verbose",
            test_fixtures("test.pdf"), testpath/"out.dcm"
-    File.exist? testpath/"out.dcm"
+    system bin/"dcmftest", testpath/"out.dcm"
```

Comment:
It gave a higher certainty that the output file was in a valid format instead of just existing.